.changeset/sour-donuts-go.md

```diff
@@ -0,0 +1,25 @@
+---
+"@spectrum-css/meter": major
+"@spectrum-css/progressbar": major
+---
+
+### S2 Meter Migration
+
+This migration updates the meter component and the progress bar component. Both needed the track to have a full corner radius to make the rounded look. The progress bar's HTML template now includes the help text component but it is only used in the meter.
+
+#### New updates
+
+- Includes `--sizeM` and `--sizeXL`
+- Includes help text component for more context
+- Shares progress bar's field label typography
+- Added help text visual test
+
+#### Removed mods
+
+- `--mod-meter-thickness`
+- `--mod-meter-font-size`
+- `--mod-meter-spacing-top-to-text`
```

Comment:
I think the removed mods here are actually the progressbar ones: `--mod-progressbar-thickness`, etc. They're actually not removed because they can still be used, we're just not passing anything through them anymore. Any thoughts on a good way to handle/document that? 🤔